jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ErrorHandler.java

```diff
@@ -163,6 +163,10 @@ protected void generateResponse(Request request, Response response, int code, St
         callback.succeeded();
     }
 
+    protected Type defaultResponseMimeType() {
+        return Type.TEXT_HTML;
+    }
+
```

Comment:
I don't mind the idea of being able to configure a default response content type.
Please:
1. Make a field `defaultResponseContentType`, with `public` getters and setters (no there is no need to extend `ErrorHandler`.
2. Write a test case in `org.eclipse.jetty.server.ErrorHandlerTest`.

Let us know if you are willing to do the above.
Thanks!

Reply:
@sbordet I applied & pushed the changes